cmd/gitserver/Dockerfile

```
@@ -38,7 +38,9 @@ LABEL com.sourcegraph.github.url=https://github.com/sourcegraph/sourcegraph/comm
 
 RUN apk add --no-cache \
     # We require git 2.34.1 because we use git-repack with flag --write-midx.
-    'git>=2.34.1' --repository=http://dl-cdn.alpinelinux.org/alpine/v3.15/main  \
+    # We require git 2.35.2 to fix this vulnerability:
```

I'd delete the old comment (it's 4 minor versions behind, I think we don't need to call this out) and I'd change this one here to not mention 2.35.2 (it's confusing that's it's named, but then we don't use it:

```suggestion
    # We require git 2.38+ because we need a fix for this vulnerability to be included:
```